lib/cacheable/model_extensions.rb

```diff
@@ -0,0 +1,18 @@
+module Cacheable
+  module ModelExtensions
+    def self.included(base)
```

Comment:
why not use `AS::Concern` ?

Reply:
Just to no bother to add another non-required but required dependency to the gem. Active Support is not a dependency of the gem, so we are relying on the application to include it. Since this is so simple, I preferred to just implement it.

Reply:
> Active Support is not a dependency of the gem

🤔 But since you are making use of it in the Railtie, then it is an implicit dependency ?

Reply:
Yes. I just didn't want to bother to use Active Support since it was not needed.